axon-server-connector/src/main/java/org/axonframework/axonserver/connector/processor/EventProcessorController.java

```diff
@@ -29,19 +30,19 @@
  */
 public class EventProcessorController {
 
-    private final EventHandlingConfiguration eventHandlingConfiguration;
+    private final EventProcessingConfiguration eventProcessingConfiguration;
 
     private final Deque<Consumer<String>> pauseHandlers = new ArrayDeque<>();
 
     private final Deque<Consumer<String>> startHandlers = new ArrayDeque<>();
 
-    public EventProcessorController(EventHandlingConfiguration eventHandlingConfiguration) {
-        this.eventHandlingConfiguration = eventHandlingConfiguration;
+    public EventProcessorController(EventProcessingConfiguration eventProcessingConfiguration) {
+        this.eventProcessingConfiguration = eventProcessingConfiguration;
     }
 
     public EventProcessor getEventProcessor(String processorName){
-        return this.eventHandlingConfiguration
-                .getProcessor(processorName)
+        return this.eventProcessingConfiguration
```

Comment:
Should we change this to make it an Optional<EventProcessor>? I don't like throwing RuntimeExceptions directly.
(I do realize this was already the case)

Reply:
I can change this, but a lot of things depend on the presence of a certain processor. I would go for an exceptional case here instead of silent failure. Would like to hear @saratry / @MGathier opinions on this.